packages/serverless/src/domain/use-case/merkle-tree.ts

```diff
@@ -1,166 +1,174 @@
+    const rightNodeField = await modelMerkleTree.getNode(
+      childrenLevel,
+      rightChildIndex,
+      currentDate
     );
-  }
 
-  if (parentLevel <= 0) {
-    throw new Error(
-      `Invalid parentLevel: ${parentLevel}. Leaves do not have children nodes.`
-    );
+    const listZeroNode = modelMerkleTree.getListZeroNode();
+
+    return [
+      {
+        hash: leftNodeField.toString(),
+        index: leftChildIndex,
+        level: childrenLevel,
+        empty: listZeroNode[childrenLevel].equals(leftNodeField).toBoolean(),
+      },
+      {
+        hash: rightNodeField.toString(),
+        index: rightChildIndex,
+        level: childrenLevel,
+        empty: listZeroNode[childrenLevel].equals(rightNodeField).toBoolean(),
+      },
+    ];
   }
 
-  const modelMerkleTree = await ModelMerkleTree.load(databaseName);
+  public static async merkleProofPath(databaseName: string, docId: string) {
+    const modelDocumentMetadata = new ModelMetadataDocument(databaseName);
 
-  if (parentLevel >= modelMerkleTree.height) {
-    throw new Error(
-      `Invalid parentLevel: ${parentLevel}. The Merkle Tree has a height of ${modelMerkleTree.height}.`
-    );
-  }
-
-  const currentDate = new Date();
-
-  const childrenLevel = parentLevel - 1;
-  const leftChildIndex = parentIndex * 2n;
-  const rightChildIndex = leftChildIndex + 1n;
-
-  const leftNodeField = await modelMerkleTree.getNode(
-    childrenLevel,
-    leftChildIndex,
-    currentDate
-  );
-  const rightNodeField = await modelMerkleTree.getNode(
-    childrenLevel,
-    rightChildIndex,
-    currentDate
-  );
-
-  const zeroNodes = modelMerkleTree.getZeroNodes();
-
-  return [
-    {
-      hash: leftNodeField.toString(),
-      index: Number(leftChildIndex),
-      level: childrenLevel,
-      empty: zeroNodes[childrenLevel].equals(leftNodeField).toBoolean(),
-    },
-    {
-      hash: rightNodeField.toString(),
-      index: Number(rightChildIndex),
-      level: childrenLevel,
-      empty: zeroNodes[childrenLevel].equals(rightNodeField).toBoolean(),
-    },
-  ];
-}
+    const docMetadata = await modelDocumentMetadata.findOne({
+      docId,
+    });
 
-export async function getMerkleWitnessPath(
-  databaseName: string,
-  docId: string
-) {
-  const modelDocumentMetadata = new ModelMetadataDocument(databaseName);
+    if (!docMetadata) {
+      throw Error(`Metadata has not been found`);
+    }
 
-  const docMetadata = await modelDocumentMetadata.findOne({
-    docId,
-  });
+    const merkleTree = await ModelMerkleTree.getInstance(databaseName);
 
-  if (!docMetadata) {
-    throw Error(`Metadata has not been found`);
+    return merkleTree.getMerkleProofPath(
+      BigInt(docMetadata.merkleIndex),
+      new Date()
+    );
   }
-
-  const merkleTree = await ModelMerkleTree.load(databaseName);
-
-  return merkleTree.getWitnessPath(BigInt(docMetadata.merkleIndex), new Date());
 }
+
+export default { MerkleTree };
```

Comment:
If you do this the default import will be an Object and need to .MerkleTree again to use it
<img width="403" alt="image" src="https://github.com/user-attachments/assets/d06e3905-540f-4456-a72e-3b2e3a032f86" />

just `export default MerkleTree`